docs/System.md

```diff
@@ -0,0 +1,93 @@
+# System Overview
+
+This document provides an overview of the Exo compilation process, as illustrated in Figure 1 of the PLDI'22 paper.
+
+![System overview](images/system-overview.png)
+
+The Exo compiler consists of a frontend and a backend, with user schedules applied in between. The input to the compiler is a set of Exo source files (`*.py`), and the output is generated C code (`*.c`).
+
```

Comment:
I feel like this doc would have made most sense if we started out with an overview of the different IRs and how they are used. In particular that LoopIR is the "main" IR used for scheduling (in between the frontend i.e. UAST->LoopIR and the backend i.e. LoopIR->C).